src/Forms/Schema/FormSchema.php

```diff
@@ -112,9 +113,29 @@ public function getSchema(Form $form)
             $schema['fields'][] = $field->getSchemaData();
         }
 
+        // Validate there are react components for all fields
+        // Note 'actions' (FormActions) are always valid because FormAction.schemaComponent has a default value
+        $this->recursivelyValidateSchemaData($schema['fields']);
+
         return $schema;
     }
 
+    private function recursivelyValidateSchemaData(array $schemaData)
+    {
+        foreach ($schemaData as $data) {
+            if (!$data['schemaType'] && !$data['component']) {
+                $name = $data['name'];
+                $message = "Could not find a react component for $name."
```

Comment:
```suggestion
                $message = "Could not find a react component for field '$name'."
```
If I have for example a gridfield named "Pages" it used to say "could not find a react component for Pages".... well, pages don't have a react component.
But now it will say "Could not find a react component for field 'Pages'", which makes it way clearer that it's not `SiteTree` it was trying to find a react component for - but the form field which has the name "Pages"

Reply:
Updated